File: levelledmobs-plugin-bukkit/src/main/java/me/lokka30/levelledmobs/bukkit/integrations/type/MobOwner.java
package me.lokka30.levelledmobs.bukkit.integrations.type;

import me.lokka30.levelledmobs.bukkit.utils.TriState;
import org.bukkit.entity.LivingEntity;
import org.jetbrains.annotations.NotNull;

@SuppressWarnings("unused")
public interface MobOwner {

    @NotNull
    TriState ownsMob(final @NotNull LivingEntity entity);

}

File: levelledmobs-plugin-bukkit/src/main/kotlin/me/lokka30/levelledmobs/bukkit/integrations/type/MobOwner.kt
package me.lokka30.levelledmobs.bukkit.integrations.type

import me.lokka30.levelledmobs.bukkit.utils.TriState
import org.bukkit.entity.LivingEntity

/*
FIXME Comment
 */
interface MobOwner {

    /*
    FIXME Comment
     */
    fun ownsMob(mob: LivingEntity): TriState

}